app/forms/candidate_interface/other_qualification_type_form.rb

```diff
@@ -25,7 +25,7 @@ class OtherQualificationTypeForm
     attr_accessor :non_uk_qualification_type
 
     validates :qualification_type, presence: true
-    validates :qualification_type, inclusion: { in: ALL_VALID_TYPES, allow_blank: false }
+    validates :qualification_type, inclusion: { in: ALL_VALID_TYPES, allow_blank: false }, unless: :no_other_qualification?
```

Comment:
This could just be something like:

```
validates :qualification_type, inclusion: { in: ALL_VALID_TYPES + ['no_other_qualification'], allow_blank: false }
```